lib/TopSitesFeed.jsm

```diff
@@ -599,6 +602,9 @@ this.TopSitesFeed = class TopSitesFeed {
     if (searchTopSite) {
       toPin.searchTopSite = searchTopSite;
     }
+    if (tippyTopIcon) {
+      toPin.tippyTopIcon = tippyTopIcon;
```

Comment:
To support pinning of sponsored top sites, we are making spoc site as orgainc site and saving its third party image url  in browser.newtab.pinned pref. This raises concern as we shouldn't be supporting pinned image url for-ever. Options here are:

a) Treat pinned spoc site as any other organic site that shows image as screenshot image url. This might look as weird as after pinning image icon will change to screenshot image.

b) Not support pin and drag and drop of spoc item and have it show up fixed in topsites for experiments

@wolasi including for help clarify this flow. Thanks

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1575063

We have a delay behavior that prevents us from using custom images (exists in master already). The bug only shows custom images as icons after a hard browser reset. Digging into this bug and if we can fix this then we can keep the original expected behavior instead of the alternative treatment above. Hopefully can have a response end of day.

Reply:
I think a) is the better option. Why can't we use the image from adzerk as the image when it becomes organic?